components/markdown_image_expand/index.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {bindActionCreators, Dispatch} from 'redux';
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {GlobalState} from 'mattermost-redux/types/store';
+import {connect} from 'react-redux';
+
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {toggleInlineImageVisibility} from 'actions/post_actions';
+import {isInlineImageVisible} from 'selectors/posts';
+
+import MarkdownImageExpand, {Props} from './markdown_image_expand';
+
+const mapStateToProps = (state: GlobalState, ownProps: Props) => {
```

Comment:
Could we call `ownProps` just `props`? I /think/ the pattern for `ownProps` only comes into effect when defining a callback and needing to distinguish the two closures.

Reply:
Kinda fixed.